pom.xml

```diff
@@ -327,6 +494,26 @@
 
                     </executions>
                 </plugin>
+
+                <plugin>
+                    <artifactId>maven-assembly-plugin</artifactId>
+                    <version>${maven-assembly-plugin.version}</version>
+                    <executions>
+                        <execution>
+                            <phase>package</phase>
+                            <goals>
+                                <goal>single</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <attach>false</attach>
+                        <descriptors>
+                            <descriptor>assembly.xml</descriptor>
+                        </descriptors>
+                    </configuration>
+                </plugin>
```

Comment:
I am not quit sure whether we should pop plugin version to parent `pom.xml`. It only belong to `seatunnel-dist` @CalvinKirs

Reply:
I need to re-modify the packaging method later, and shade is not a good way